tests/v2/validation/nodescaling/scaling_nodepools.go

```diff
@@ -0,0 +1,57 @@
+package scaling
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/machinepools"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioninginput"
+	rke1 "github.com/rancher/rancher/tests/framework/extensions/rke1/nodepools"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	ProvisioningSteveResourceType = "provisioning.cattle.io.cluster"
+	defaultNamespace              = "fleet-default"
+)
+
+func ScalingRKE2K3SNodePools(t *testing.T, client *rancher.Client, clusterID string, clustersConfig *provisioninginput.Config) {
+	cluster, err := client.Steve.SteveType(ProvisioningSteveResourceType).ByID(clusterID)
+	require.NoError(t, err)
+
+	clustersConfig.NodesAndRoles[0].Quantity = clustersConfig.NodesAndRoles[0].Quantity + 1
+	logrus.Infof("Scaling the node pool up to %v nodes", clustersConfig.NodesAndRoles[0].Quantity)
+	clusterResp, err := machinepools.ScaleMachinePoolNodes(client, cluster, clustersConfig.NodesAndRoles[0])
+	require.NoError(t, err)
+
+	err = pods.VerifyReadyDaemonsetPods(t, client, cluster)
+	require.NoError(t, err)
+
+	clustersConfig.NodesAndRoles[0].Quantity = clustersConfig.NodesAndRoles[0].Quantity - 1
+	logrus.Infof("Scaling the node pool down to %v nodes", clustersConfig.NodesAndRoles[0].Quantity)
+	scaledClusterResp, err := machinepools.ScaleMachinePoolNodes(client, clusterResp, clustersConfig.NodesAndRoles[0])
+	require.NoError(t, err)
+
+	err = pods.VerifyReadyDaemonsetPods(t, client, scaledClusterResp)
+	require.NoError(t, err)
+}
+
+func ScalingRKE1NodePools(t *testing.T, client *rancher.Client, clusterID string, clustersConfig *provisioninginput.Config) {
+	cluster, err := client.Management.Cluster.ByID(clusterID)
+	require.NoError(t, err)
+
+	node, err := rke1.AllNodeList(client, cluster, clustersConfig.NodesAndRolesRKE1[0])
+	require.NoError(t, err)
+
+	clustersConfig.NodesAndRolesRKE1[0].Quantity = clustersConfig.NodesAndRolesRKE1[0].Quantity + 1
+	logrus.Infof("Scaling the node pool up to %v nodes", clustersConfig.NodesAndRolesRKE1[0].Quantity)
+	_, err = rke1.ScaleNodePoolNodes(client, cluster, node, clustersConfig.NodesAndRolesRKE1[0])
+	require.NoError(t, err)
+
+	clustersConfig.NodesAndRolesRKE1[0].Quantity = clustersConfig.NodesAndRolesRKE1[0].Quantity - 1
```

Comment:
I think this should just be `-clustersConfig.NodesAndRolesRKE1[0].Quantity` instead of -1

Reply:
But that's just going to keep the scaled nodepool at the same number instead of scaling it down. Unsure what you mean here?